ide/faqs.md

```diff
@@ -45,3 +45,16 @@ Yes, this is totally normal. If curious, `~/workspace/.c9/` is used by Cloud9 ID
 Generally, you shouldn't touch this folder or its contents. Files and folders whose names start with a dot (aka dot files) should be hidden by default. You may want to hide the `.c9` folder (along with any other dot files) by clicking the gear icon atop the file browser on the left and unchecking **Show Hidden Files**.
 
 ![show hidden files](hidden-files.png)
+
+## My workspace has less than 5GB of total disk space. How to get a new workspace with 5GB disk space?
```

Comment:
Probably better as "What if my workspace runs out of space?" or such, since students won't necessarily know that 5GB is the limit.